pyensembl/__init__.py

```diff
@@ -19,6 +19,26 @@
 from .gene import Gene
 from .gtf import GTF
 from .locus import Locus
+from .release_info import check_release_number
 from .search import find_nearest_locus
 from .sequence_data import SequenceData
 from .transcript import Transcript
+
+_cache = {}
+
+def cached_release(version, auto_download=True):
+    """Cached construction of EnsemblRelease objects. It's desirable to reuse
+    the same EnsemblRelease object since each one will store a lot of cached
+    annotation data in-memory.
+    """
+    version = check_release_number(version)
+    key = version, auto_download
+    if key not in _cache:
+        ensembl = EnsemblRelease(version, auto_download=auto_download)
+        _cache[key] = ensembl
+    return _cache[key]
+
+ensembl_grch36 = ensembl54 = cached_release(54)  # last release for GRCh36/hg18
+ensembl_grch37 = ensembl75 = cached_release(75)  # last release for GRCh37/hg19
```

Comment:
No, it will only download anything if you start using the methods on these objects.